### ISSUE TITLE
option to not render next/prev buttons

### ISSUE THREAD

**Issue by sKopheK** [jonelantha/react-responsive-pagination#62]
hi, any plans for prop to remove next/prev btns?
if i hide them using CSS, JS won't calculate correct width because the plugin works the way it renders all different buttons first (single digit page btn as first child) and as first and last are hidden to achieve desired looks, btn fitting this condition has width of 0

**Comment by jonelantha** [comment]
Yes it's best to add a prop for this, hopefully this will be added to a future release

**Comment by sKopheK** [comment]
my 2 cents https://github.com/jonelantha/react-responsive-pagination/pull/63
